src/shared-components/alert/index.tsx

```diff
@@ -92,17 +95,17 @@ export const Alert: Alert = (alertProps) => {
       return;
     }
 
-    timer = window.setTimeout(
```

Comment:
`Assignments to the 'timer' variable from inside React Hook useEffect will be lost after each render. To preserve the value over time, store it in a useRef Hook and keep the mutable value in the '.current' property. Otherwise, you can move this variable directly inside useEffect.`